manifests/init.pp

```diff
@@ -244,6 +254,9 @@
     if $package_corosync {
       fail('$corosync::package_corosync and $corosync::packages must not be mixed!')
     }
+    if $packageopts_corosync {
+      fail('$corosync::packageopts_corosync and $corosync::packages must not be mixed!')
```

Comment:
Hi, in Line 245 is an "if" for a deprecated variable $packages, thats why I added this fails, like it was done for the other "new-style package parameters".